README.md

```diff
@@ -60,6 +60,16 @@ Folder structure on root directory:
 |           +-- Layout1.js
 |       +-- _Layout2
 |           +-- Layout2.js
++-- _pages
+|   +-- _Home
+|       +-- home.js
+|       +-- home.scss
+|   +-- _About
+|       +-- about.js
+|       +-- about.scss
+|   +-- _NotFound
+|       +-- notFound.js
+|       +-- notFound.scss
```

Comment:
Since this is just a explanatory preview of the structure, I think its best to show a generic folder and files. 
eg.  
```
+-- _pages
      +-- _SomePage
           +-- SomePage.js
           +-- SomePage.css
```
You can change the cypress structure to reflect the same logic.

Reply:
Implemented the feedback